tool/generate_bundles.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Runs `mason bundle` to generate bundles for all bricks within the respective templates directories.
+
+bricks=(very_good_core very_good_dart_package very_good_dart_cli very_good_flutter_package very_good_flutter_plugin)
```

Comment:
we could have a cli for this lol

Reply:
Better yet, we could have a build generator package for this (reusable).

Reply:
it could also be its own brick 🤣